### PR TITLE
Add Akinator

### PR DESCRIPTION
**Pull request by alrjunior**

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -705,6 +705,17 @@
...
     },
 
+    {
+        "name": "Akinator",
+        "url": "https://akinator.com",
+        "difficulty": "easy",
+        "notes": "At the app ⮕ Click in settings (gear icon) ⮕ Account settings ⮕ Delete account (at the bottom of the screen).",
+        "notes_pt_br": "No aplicativo ⮕ Clique em configurações (ícone da engrenagem) ⮕ Configurações de conta ⮕ Excluir conta (no fim da tela).",
+        "domains": [
+            "akinator.com"
```
Comment (Kefaku) [inline _data/sites.json:715]: There are more localized/mobile domains.

```suggestion
            "akinator.com",
            "ar.akinator.com",
            "cn.akinator.com",
            "de.akinator.com",
            "en.akinator.com",
            "es.akinator.com",
            "fr.akinator.com",
            "id.akinator.com",
            "il.akinator.com",
            "it.akinator.com",
            "jp.akinator.com",
            "kr.akinator.com",
            "nl.akinator.com",
            "pl.akinator.com",
            "pt.akinator.com",
            "ru.akinator.com",
            "tr.akinator.com",
            "akinator.mobi",
            "ar.akinator.mobi",
            "cn.akinator.mobi",
            "de.akinator.mobi",
            "en.akinator.mobi",
            "es.akinator.mobi",
            "fr.akinator.mobi",
            "id.akinator.mobi",
            "il.akinator.mobi",
            "it.akinator.mobi",
            "jp.akinator.mobi",
            "kr.akinator.mobi",
            "nl.akinator.mobi",
            "pl.akinator.mobi",
            "pt.akinator.mobi",
            "ru.akinator.mobi",
            "tr.akinator.mobi"
```

**Review by Kefaku — CHANGES_REQUESTED** [review]

File: _data/sites.json [hunk 2]
```diff
@@ -705,6 +705,50 @@
         ]
     },
 
+    {
+        "name": "Akinator",
+        "url": "https://akinator.com",
+        "difficulty": "easy",
+        "notes": "At the app ⮕ Click in settings (gear icon) ⮕ Account settings ⮕ Delete account (at the bottom of the screen).",
```
Comment (tupaschoal) [inline _data/sites.json:712]:
```suggestion
        "notes": "At the app, click on settings (gear icon) ⮕ Account settings ⮕ Delete account (at the bottom of the screen).",
```

File: _data/sites.json [hunk 3]
```diff
@@ -705,6 +705,50 @@
...
+        "notes_pt_br": "No aplicativo ⮕ Clique em configurações (ícone da engrenagem) ⮕ Configurações de conta ⮕ Excluir conta (no fim da tela).",
```
Comment (tupaschoal) [inline _data/sites.json:713]:
```suggestion
        "notes_pt_br": "No aplicativo, clique em configurações (ícone da engrenagem) ⮕ Configurações de conta ⮕ Excluir conta (no fim da tela).",
```

**Review by tupaschoal — APPROVED** [review]

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**